### PR TITLE
[Merged by Bors] - Fix comment typo

### PR DESCRIPTION
**Pull request by eiei114**
# Objective

- Fix some typos

## Solution

For the first time in my life, I made a pull request to OSS.
Am I right?

### CODE REVIEW

File: crates/bevy_gltf/src/lib.rs [hunk 1]
```diff
@@ -57,7 +57,7 @@ pub struct GltfNode {
     pub transform: bevy_transform::prelude::Transform,
 }
 
-/// A glTF mesh, which may consists of multiple [`GtlfPrimitives`](GltfPrimitive).
+/// A glTF mesh, which may consists of multiple [`GltfPrimitives`](GltfPrimitive).
```
Comment (rparrett) [inline crates/bevy_gltf/src/lib.rs:60]:
```suggestion
/// A glTF mesh, which may consist of multiple [`GltfPrimitives`](GltfPrimitive).
```

File: crates/bevy_ui/src/focus.rs [hunk 2]
```diff
@@ -25,7 +25,7 @@ use smallvec::SmallVec;
 /// If a UI node has both [`Interaction`] and [`ComputedVisibility`] components,
 /// [`Interaction`] will always be [`Interaction::None`]
 /// when [`ComputedVisibility::is_visible()`] is false.
-/// This ensures that hidden UI nodes are not interactable,
+/// This ensures that hidden UI nodes are not intractable,
```
Comment (rparrett) [inline crates/bevy_ui/src/focus.rs:28]: This is "interactable" as in "able to be interacted with" which is not quite a dictionary word but is used very commonly.

File: crates/bevy_ui/src/focus.rs [hunk 3]
```diff
@@ -127,7 +127,7 @@ pub fn ui_focus_system(
         .iter_mut()
         .filter_map(
             |(entity, node, global_transform, interaction, focus_policy, clip, visibility)| {
-                // Nodes that are not rendered should not be interactable
+                // Nodes that are not rendered should not be intractable
```
Comment (rparrett) [inline crates/bevy_ui/src/focus.rs:130]: ditto

File: crates/bevy_utils/src/lib.rs [hunk 4]
```diff
@@ -194,7 +194,7 @@ impl Hasher for PassHasher {
     }
 }
 
-/// A [`HashMap`] pre-configured to use [`Hashed`] keys and [`PassHash`] passthrough hashing.
+/// A [`HashMap`] pre-configured to use [`Hashed`] keys and [`PassHash`] pass through hashing.
```
Comment (rparrett) [inline crates/bevy_utils/src/lib.rs:197]: I think this one is okay as-is as well.

**Review by rparrett — CHANGES_REQUESTED** [review]
Ooh, couple of good catches. I found an additional typo in one of the lines you modified.

But I think a few of the changes aren't necessary.

**Comment by eiei114** [comment]
@rparrett 
The problem was that English was not my first language and I did it messily using the editor's proofreading function.

Thank you for the detailed instructions🥰

**Comment by alice-i-cecile** [comment]
Agree with the comments from @rparett :) @eiei114, thanks, and welcome to OSS 🎉  On the "Files changed" tab you can commit suggestions directly :)

**Comment by rparrett** [comment]
Oops, I didn't add "github suggestions" for the comments where things were okay as-is. So you'll want to:

- Pull the suggestion commit that github made in your branch for you to your local branch
- Make those other changes manually
- Push the branch back

File: crates/bevy_ui/src/focus.rs (hunk 2 above)
Comment (alice-i-cecile) [reply crates/bevy_ui/src/focus.rs:28]:
```suggestion
/// This ensures that hidden UI nodes are not interactable,
```

File: crates/bevy_ui/src/focus.rs (hunk 3 above)
Comment (alice-i-cecile) [reply crates/bevy_ui/src/focus.rs:130]:
```suggestion
                // Nodes that are not rendered should not be interactable
```

File: crates/bevy_utils/src/lib.rs (hunk 4 above)
Comment (alice-i-cecile) [reply crates/bevy_utils/src/lib.rs:197]:
```suggestion
/// A [`HashMap`] pre-configured to use [`Hashed`] keys and [`PassHash`] passthrough hashing.
```

**Comment by alice-i-cecile** [comment]
I've added suggestions for those reversions now, in case that's easier for you :) You can "add suggestions to batch", and get a slightly nicer commit history too. It's not a big deal here though, this is a tiny PR and we squash commits on merge.

**Comment by eiei114** [comment]
Sorry🤦‍♂️ I forgot that you only raised the suggestion and no changes were made 😂

**Review by rparrett — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Looks great! You're now *officially* a contributor to Bevy ;)

bors r+

**Pull request closed without merging**